core/editor/editor.php

```diff
@@ -972,4 +820,36 @@ private function bc_move_document_filters() {
 	public function set_post_id( $post_id ) {
 		$this->post_id = $post_id;
 	}
+
+	/**
+	 * @return Loading_Strategy_Interface
+	 */
+	private function make_loading_strategy() {
+		$is_editor_v2_active = Plugin::$instance->experiments->is_feature_active( 'editor_v2' );
+
+		// Nonce verification is not required, using param from routing purpose.
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		if ( $is_editor_v2_active && isset( $_GET['v'] ) && '2' === $_GET['v'] ) {
+			return new Editor_V2_Loading_Strategy();
+		}
+
+		return new Editor_V1_Loading_Strategy();
+	}
+
+	/**
+	 * Adding Editor V2 experiment.
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	private function register_editor_v2_experiment() {
+		Plugin::$instance->experiments->add_feature( [
+			'name' => 'editor_v2',
+			'title' => esc_html__( 'Editor V2', 'elementor' ),
+			'description' => esc_html__( 'Enable the new editor.', 'elementor' ),
+			'hidden' => true,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+			'status' => Experiments_Manager::RELEASE_STATUS_ALPHA,
+		] );
+	}
```

Comment:
Move to a module

Reply:
Why it should be in a module? is this logic not the editor's core?